### ISSUE TITLE
test on testing set

### ISSUE THREAD

**Issue by Ali-Bakhtiarian** [2gunsu/monocon-pytorch#25]
Hi, when I run the test.py it just evaluate the .pth file on the evaluation images that derived from training set. How can I test the checkpoint on main testing set (7518 .png images)?

**Comment by 2gunsu** [comment]
Hello.
Thank you for your interest in the code.

Unfortunately, the current implementation version does not yet support inference from the test set. 😢 The complete version needs to be implemented quickly, but it's been delayed because I'm busy with work right now.

I will check if it can be modified in the implementation and reply to you again.
Thank you.

**Comment by Ali-Bakhtiarian** [comment]
Hello!
Thank you for your response.
I am currently working on your code for my Master's thesis, and I needed this part urgently for testing. I have successfully generated a text file for the validation set but encountered an error while testing on the test set. I will upload the completed work as a branch of your GitHub project so that others can also benefit from it.
Thank you.

**Comment by 2gunsu** [comment]
Thank you :)
I hope your work will be successfully completed. 
I'll be rooting for you! 🤗

**Comment by zzy0731** [comment]
@Ali-Bakhtiarian  how do you successfully generate text file for the validation set. Where do you modify  the code ?

**Comment by moyanouo** [comment]
> 你好。 感谢您对代码的兴趣。
> 
> 不幸的是，当前的实现版本尚不支持从测试集进行推理。 😢 完整版本需要尽快实现，但是因为我现在工作比较忙所以推迟了。
> 
> 我会在实施中检查是否可以修改并再次回复您。 谢谢。

hello？
